azext_iot/common/utility.py

```diff
@@ -134,7 +138,7 @@ def process_json_arg(content, argument_name, preserve_order=False):
             )
 
         file_content_error = "from file: '{}' ".format(json_from_file)
-        raise CLIError(
+        raise CLIInternalError(
             "Failed to parse json {}for argument '{}' with exception:\n    {}".format(
```

Comment:
nit: can this typo be fixed

Reply:
What is the typo? I'm not seeing it right now.

Reply:
The string format is not a typo...look at the inputs to the placeholder, it is either the value of `file_content_error` which ends in a space, or the empty string.